guide/contribute/formatting.md

```diff
@@ -0,0 +1,407 @@
+    - level 3 item (ul)
+- level 1 item (ul)
+
+#### Task list
+
+```markdown
+- [ ] Hello, this is a TODO item
+- [ ] Hello, this is another TODO item
+- [x] Goodbye, this item is done
+```
+- [ ] Hello, this is a TODO item
+- [ ] Hello, this is another TODO item
+- [x] Goodbye, this item is done
+
+#### Definition with HTML syntax.
+
+```markdown
+<dl>
+<dt>Name</dt>
+<dd>Godzilla</dd>
+<dt>Born</dt>
+<dd>1952</dd>
+<dt>Birthplace</dt>
+<dd>Japan</dd>
+<dt>Color</dt>
+<dd>Green</dd>
+</dl>
+```
+<dl>
+<dt>Name</dt>
+<dd>Godzilla</dd>
+<dt>Born</dt>
+<dd>1952</dd>
+<dt>Birthplace</dt>
+<dd>Japan</dd>
+<dt>Color</dt>
+<dd>Green</dd>
+</dl>
+
+### Tables
+
+```markdown
+| head1        | head two          | three |
+|:-------------|:------------------|:------|
+| ok           | good swedish fish | nice  |
+| out of stock | good and plenty   | nice  |
+| ok           | good `oreos`      | hmm   |
+| ok           | good `zoute` drop | yumm  |
+```
+
+| head1        | head two          | three |
+|:-------------|:------------------|:------|
+| ok           | good swedish fish | nice  |
+| out of stock | good and plenty   | nice  |
+| ok           | good `oreos`      | hmm   |
+| ok           | good `zoute` drop | yumm  |
+
+### Code embedding
+
+```js
+// Javascript code with syntax highlighting.
+var fun = function lang(l) {
+  dateformat.i18n = require('./lang/' + l)
+  return true;
+}
+```
+
+```ruby
+# Ruby code with syntax highlighting
+GitHubPages::Dependencies.gems.each do |gem, version|
+  s.add_dependency(gem, "= #{version}")
+end
+```
+
+
+### There's a horizontal rule below this.
+```markdown
+* * *
+```
+* * *
+
+### Embedding content from external sources
+
+#### YouTube video embed
+
+```markdown
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/zKMRSLbQEqk/mqdefault.jpg)](https://www.youtube.com/watch?v=zKMRSLbQEqk)
+```
+[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/zKMRSLbQEqk/mqdefault.jpg)](https://www.youtube.com/watch?v=zKMRSLbQEqk)
+
+#### Figma embed
+
+Figma embeds are automatically resized to comfortable fit into the visible screen area.
+
+```html
+<div class="figma-embed">
+<iframe width="450" height="1000" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FHggAJoHhLXPH0oZQEr1D4D%2FBitcoin-Design-Guide%3Fnode-id%3D166%253A0%26viewport%3D1714%252C3489%252C1%26scaling%3Dmin-zoom" allowfullscreen></iframe>
+</div>
+```
+
+<div class="figma-embed">
+<iframe width="450" height="1000" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FHggAJoHhLXPH0oZQEr1D4D%2FBitcoin-Design-Guide%3Fnode-id%3D166%253A0%26viewport%3D1714%252C3489%252C1%26scaling%3Dmin-zoom" allowfullscreen></iframe>
+</div>
+
+```
+Long, single-line code blocks should not wrap. They should horizontally scroll if they are too long. This line should be long enough to demonstrate this.
+```
+
+```
+The final element.
+```
+
+[^1]: https://bitcoin.design "Footnote with a caption"
```

Comment:
I think the reference for this footnote is missing.

Reply:
Where should I put it?

Reply:
Done.